pkg/webhooks/admission/jobs/validate/admit_job_test.go

```diff
@@ -1221,39 +1221,35 @@ func TestValidateJobCreate(t *testing.T) {
 		},
 	}
 
-	for _, testCase := range testCases {
-		t.Run(testCase.Name, func(t *testing.T) {
-			defaultqueue := schedulingv1beta2.Queue{
-				ObjectMeta: metav1.ObjectMeta{
-					Name: "default",
-				},
-				Spec: schedulingv1beta2.QueueSpec{
-					Weight: 1,
-				},
-				Status: schedulingv1beta2.QueueStatus{
-					State: schedulingv1beta2.QueueStateOpen,
-				},
-			}
-			// create fake volcano clientset
-			config.VolcanoClient = fakeclient.NewSimpleClientset()
-			informerFactory := informers.NewSharedInformerFactory(config.VolcanoClient, 0)
-			queueInformer := informerFactory.Scheduling().V1beta1().Queues()
-			config.QueueLister = queueInformer.Lister()
+	defaultqueue := &schedulingv1beta2.Queue{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "default",
+		},
+		Spec: schedulingv1beta2.QueueSpec{
+			Weight: 1,
+		},
+		Status: schedulingv1beta2.QueueStatus{
+			State: schedulingv1beta2.QueueStateOpen,
+		},
+	}
 
-			stopCh := make(chan struct{})
-			informerFactory.Start(stopCh)
-			for informerType, ok := range informerFactory.WaitForCacheSync(stopCh) {
-				if !ok {
-					panic(fmt.Errorf("failed to sync cache: %v", informerType))
-				}
-			}
+	// create fake volcano clientset
+	config.VolcanoClient = fakeclient.NewSimpleClientset(defaultqueue)
+	informerFactory := informers.NewSharedInformerFactory(config.VolcanoClient, 0)
+	queueInformer := informerFactory.Scheduling().V1beta1().Queues()
+	config.QueueLister = queueInformer.Lister()
 
-			//create default queue
-			_, err := config.VolcanoClient.SchedulingV1beta1().Queues().Create(context.TODO(), &defaultqueue, metav1.CreateOptions{})
```

Comment:
createing `defaultqueue` will be removed?

Reply:
`config.VolcanoClient = fakeclient.NewSimpleClientset(defaultqueue)` has already set the queue resource.